aspects/aspects_test.go

```diff
@@ -1074,3 +1074,80 @@ func (s *aspectSuite) TestBadRequestPaths(c *C) {
 		c.Assert(err.Error(), Equals, fmt.Sprintf(`cannot parse Get request: %s`, tc.errMsg), cmt)
 	}
 }
+
+func (s *aspectSuite) TestSetMatchesOnPrefixes(c *C) {
+	databag := aspects.NewJSONDataBag()
+	aspectBundle, err := aspects.NewAspectBundle("acc", "bundle", map[string]interface{}{
+		"foo": []map[string]string{
+			{"request": "a", "storage": "a", "access": "write"},
+			{"request": "a.b", "storage": "b", "access": "write"},
```

Comment:
with this request we would try to write .b of value into b of storage? but that doesn't seem to be what happens, as value here is a not map this wouldn't work anyway